lib/server/push.api.js

```diff
@@ -36,6 +36,15 @@ Push.Configure = function(options) {
     // Add debug info
     if (Push.debug) console.log('Push.Configure', options);
 
+    // config collections for project
+    var prefix = options.dbCollectionPrefix || '';
+    var notificationsCollection =  prefix + '_raix_push_notifications';
+    Push.notifications = new Mongo.Collection(notificationsCollection);
```

Comment:
You are right, but I did not move the variable, what I moved was the place to put the name, cause here is where I have access to the config